src/modules/commander/state_machine_helper.cpp

```diff
@@ -651,6 +657,7 @@ bool set_nav_state(struct vehicle_status_s *status, struct commander_state_s *in
 		/* require RC for all manual modes */
 		if (rc_loss_enabled && (status->rc_signal_lost || status_flags->rc_signal_lost_cmd) && armed) {
 			status->failsafe = true;
+			PX4_WARN(reason_no_rc);
```

Comment:
These `PX4_WARN` are already much better, what do you think about having them report to mavlink as well?
However, that might be too verbose, not sure.